examples/compilation/library.jl

```diff
@@ -0,0 +1,102 @@
+# EXCLUDE FROM TESTING
+
+using CUDArt
```

Comment:
Yes, that would be a logical step forward. That's how it used to be, in CUDAnative, before I ripped that functionality out. For CUDArt, it might also simplify the tests a bit.